examples/ase/example_equilibration.py

```
@@ -0,0 +1,175 @@
+    MaxwellBoltzmannDistribution(atoms, temperature_K=600)
+
+    # Create Langevin dynamics
+    dyn = Langevin(
+        atoms,
+        timestep=5 * units.fs,
+        temperature_K=500,
+        friction=0.01 / units.fs,
+    )
+
+    print("Starting convergence-controlled equilibration...")
+    print(f"Target temperature: 500 K")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)